.circleci/config.yml

```diff
@@ -27,9 +27,9 @@ commands:
             # we are trying to test.
             rm -Rf emscripten
             echo "import os" >> ~/.emscripten
-            echo "V8_ENGINE = os.path.expanduser('~/.jsvu/v8')" >> ~/.emscripten
+            echo "V8_ENGINE = [os.path.expanduser('~/.jsvu/v8'), '--wasm-staging']" >> ~/.emscripten
```

Comment:
Are you sure we want to do this?   This means for example if we start emitting non-trapping-floattoint instructions all our tests will continue to pass.. even though we would break users in production.

Perhaps "wasm-staging" should be another "core" variant that we run?

Reply:
Interesting, yeah, we should be careful here.

Since we test primarily on node, I think we are ok to add this here. Basically v8 is our "bleeding edge" environment anyhow.

Reply:
Ah I see.  That make sense.  LGTM.  

Maybe a comment?

Reply:
Sure, adding.